testscommon/factory/bootstrapperFactoryStub.go

```diff
@@ -0,0 +1,21 @@
+package factory
+
+import (
+	"github.com/multiversx/mx-chain-go/process"
+	"github.com/multiversx/mx-chain-go/process/sync"
+	"github.com/multiversx/mx-chain-go/testscommon"
+)
+
+// BootstrapperFactoryStub -
+type BootstrapperFactoryStub struct {
```

Comment:
This is duplicated with the one from testscommon/factory/bootstrapperFactoryMock.go.
Please merge them into a single file

Reply:
removed